micrometer-core/src/main/java/io/micrometer/core/instrument/push/PushMeterRegistry.java

```diff
@@ -121,4 +126,14 @@ public void close() {
         super.close();
     }
 
+    // VisibleForTesting
+    long calculateInitialDelay() {
+        long stepMillis = config.step().toMillis();
+        Random random = new Random();
+        long randomOffsetWithinStep = (long) (stepMillis * random.nextDouble()
+                * PERCENT_RANGE_OF_RANDOM_PUBLISHING_OFFSET);
+        long offsetToStartOfNextStep = stepMillis - clock.wallTime() % stepMillis;
```

Comment:
Can we make it more obvious?
```suggestion
        long offsetToStartOfNextStep = stepMillis - (clock.wallTime() % stepMillis);
```

Reply:
We can, but I think Lift might complain about unnecessary parentheses 😄